portfolio/src/main/webapp/contact.html

```diff
@@ -11,7 +11,7 @@
     <div class="menu">  
         <a href='index.html'><b>Home</b></a>
```

Comment:
Double quotes throughout html https://google.github.io/styleguide/htmlcssguide.html#HTML_Quotation_Marks

Use double ("") rather than single quotation marks ('') around attribute values.